src/Arcus.EventGrid.Testing/Infrastructure/Hosts/EventConsumerHost.cs

```diff
@@ -50,40 +62,17 @@ public EventConsumerHost(ILogger logger)
         /// <exception cref="ArgumentException">Thrown when the <paramref name="rawReceivedEvents"/> is blank.</exception>
         /// <exception cref="ArgumentNullException">Thrown when the <paramref name="logger"/> is <c>null</c>.</exception>
         /// <exception cref="JsonReaderException">Thrown when the <paramref name="rawReceivedEvents"/> failed to be read as valid JSON.</exception>
-        protected static void EventsReceived(string rawReceivedEvents, ILogger logger)
+        [Obsolete("Use the overload without the logger instead")]
+        protected void EventsReceived(string rawReceivedEvents, ILogger logger)
```

Comment:
Removed the `static` here bc it was wrong. Technically, this is now a breaking change. We could add some code to make sure that we still have the `static` but would love to hear what you think first. Since it's `protected`, we may overlook this as this class in any case have to be inherited from anway.

Reply:
In theory it is a breaking change, however, it's a class in a testing project.   Practically, I don't think there will be many ppl that have inherited from this class.   I mean:  the amount of people impacted will be very small.  I wouldn't put much effort in it to keep it backwards compatible.

Reply:
Ok, then we can leave this as it is.
It's indeed a testing project, for people to test out Azure stuff and assert with EventGrid events.

Ok, let's keep it like this, then. Thx for the help!